### PR TITLE
Adding PLUG COMPACT EU EM T

### PR DESCRIPTION
**Pull request by rj2s**
Working fine with ZB2MQTT but not yet supported

### CODE REVIEW

File: src/devices/ledvance.ts [hunk 1]
```diff
@@ -332,6 +332,15 @@ const definitions: DefinitionWithExtend[] = [
         description: 'SMART+ Outdoor Plug EU',
         extend: [ledvanceOnOff({powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['PLUG COMPACT EU EM T'],
+        model: 'PLUG COMPACT EU EM T',
```
Comment (Koenkk) [inline src/devices/ledvance.ts:337]: Use the model number written on the device here

File: src/devices/ledvance.ts [hunk 2]
```diff
@@ -332,6 +332,15 @@ const definitions: DefinitionWithExtend[] = [
...
+        vendor: 'LEDVANCE',
+        description: 'Automatically generated definition',
```
Comment (Koenkk) [inline src/devices/ledvance.ts:339]: Fill in a sensible description here.

File: src/devices/ledvance.ts (hunk 1 above)
Comment (rj2s) [reply src/devices/ledvance.ts:337]: Model is 4099854293276

File: src/devices/ledvance.ts (hunk 2 above)
Comment (rj2s) [reply src/devices/ledvance.ts:339]: Description should be : PLUG COMPACT EU EM T

**Comment by Koenkk** [comment]
Merged it with the already existing 4099854293276, thanks!

**Pull request merged**